pkgs/development/tools/wasm-bindgen-cli/0001-Add-cargo.lock-for-rustPlatform.buildRustPackage-in-.patch

```diff
@@ -0,0 +1,2254 @@
+From ab4953af561946fbbe3a7d6ce0c41931dcb28d4e Mon Sep 17 00:00:00 2001
+From: Maximilian Bosch <maximilian@mbosch.me>
+Date: Thu, 29 Aug 2019 21:23:14 +0200
+Subject: [PATCH] Add cargo.lock for `rustPlatform.buildRustPackage` in nixpkgs
```

Comment:
Does upstream not commit the lock file?

Reply:
No (at least I didn't find a suitable one in the repo we fetch in this expression). Or am I misunderstanding you? :)

Reply:
Understood exactly. This won't block merge, but can you submit a PR to them commiting the lock file? Kinda weird they don't have one in their github repo.

Reply:
Reported in https://github.com/rustwasm/wasm-bindgen/issues/1819.